src/libstore/unix/build/local-derivation-goal.cc

```diff
@@ -532,9 +532,10 @@ void LocalDerivationGoal::startBuilder()
 
     /* Right platform? */
     if (!parsedDrv->canBuildLocally(worker.store))
-        throw Error("a '%s' with features {%s} is required to build '%s', but I am a '%s' with features {%s}",
+        throw Error("a '%s' with features {%s} & not {%s} is required to build '%s', but I am a '%s' with features {%s}",
```

Comment:
Not pretty when empty, but that was already the case for required ones.

Reply:
Yeah, I'm curious how we should handle things when it's empty.